### ISSUE TITLE
[UX] Hide path display: improve help text and/or add permissions matrix

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4657]
Current UI:

![image](https://user-images.githubusercontent.com/2423362/94527074-214fb880-023f-11eb-9668-013d96aa9226.png)

>If enabled, this content can be placed as a block or within a listing but will not be accessible at its URL without proper permissions. This facilitates "hidden path" content that is meant to be embedded with other content and never on its own. Users who have the "View hidden paths" permission will still be able to visit the full page URL, while those without this permission will receive a "Page not found" error.

Suggestion (mockup of shorter help text + inline permission matrix):

![image](https://user-images.githubusercontent.com/2423362/94528582-45ac9480-0241-11eb-9506-260e170df8a5.png)

> Content with a "hidden path" can be placed as a block or viewed within content listings, but it will not be accessible directly at its URL. Users with the "View hidden paths" permission will still be able to visit the full page URL, while those without this permission will receive a "Page not found" error.

Alternatively, we could simply make the `the "View hidden paths" permission` bit of the help text a link to the permissions page.

**Comment by indigoxela** [comment]
Wait a minute... :wink: 

The permission to view hidden paths is site-wide and not on a per node type basis. Adding the matrix on every node type form would be pretty confusing.

The link might be a better solution. (Personally I see no need for a change, but that's a different thing.)

**Comment by olafgrabienski** [comment]
I agree, the matrix doesn't work here because the permission isn't per content type. A link to the site-wide permission would indeed be helpful. And if we do so, also the help text could be improved. (General caveat: New text must be translated again on multilingual or localized sites.)

**Comment by klonos** [comment]
> The link might be a better solution.

Agreed 👍 ...it's just that leaving this page to go to the permissions page by following a link would cause loss of any data previously changed. A link would still better then just mentioning the permission and then having people pogo-stick around the UI to find it.

**Comment by domaingood** [comment]
Agreed +1
